src/storage/mod.rs

```diff
@@ -1223,15 +1243,20 @@ impl<E: Engine> Storage<E> {
         Err(Error::InvalidCf(cf.to_owned()))
     }
 
-    fn check_key_ranges(ranges: &[KeyRange]) -> bool {
+    fn check_key_ranges(ranges: &[KeyRange], reverse: bool) -> bool {
+        // note:
+        //   if reverse, endKey is less than startKey. endKey is lowerBound.
+        //   if not reverse, endKey is greater than startKey. endKey is upperBound
```

Comment:
Can we make this note the Rustdoc for this function?

Reply:
thanks! i fixed.